### DISCUSSION TITLE
Ability that allow players to turn around a tile via button press.

### DISCUSSION THREAD

**Discussion by Daywalkrr** [AscensionGameDev/Intersect-Engine#1529] [Ideas]
**Is your feature request related to a problem? Please describe.**
When trying to aim projectiles, it gets incredibly hard to face your target. What would be nice is if you could press a button to turn your character without actually having to move.

**Describe the solution you'd like**
Simply introduce new controls for turning a player. They would just turn the entity 90 degrees.

**Additional context**

**Comment by Cheshire92** [comment]
I disagree with replacing a button for it with it doing so automatically only. That's a fine option, but sometimes I just want to spin my character around on my own volition.

**Comment by Arufonsu** [comment]
Hello, still interested on this feature?
i'm pretty sure that we aren't using **L Control** for anything right?
i implemented something on my project, based on JC's solution, which allows players to turn the direction around, but instead of a _timer_, i decided to make it work with a specific **key**, so in order to turn around, you simply **hold** the **L Control** (which locks the player from moving, but allows it to change the direction) as you turn the character around with the Arrows or WASD keys. 

video:
[![Everything Is AWESOME](https://user-images.githubusercontent.com/17498701/147608432-f2b145ec-e60e-4626-9f90-9c0a0b83782e.png)](https://s3.us-east-2.amazonaws.com/ascensiongamedev/filehost/2ff3922fd2e5a8efccba22a477d47260.mp4 "Everything Is AWESOME")

![imagen](https://user-images.githubusercontent.com/17498701/147603981-aa4919be-37d0-4494-850a-726e51dd8018.png)

I could open a PR for this, so far it works fine on my end and is not very complex to implement.
Developers would still be able to change the default keybind if they want to.

**Comment by Cheshire92** [comment]
Modifier keys will no longer be valid as a hotkey on their own if this goes through: #1016
